### ISSUE TITLE
Toast doesn't work outside of vue templates

### ISSUE THREAD

**Issue by damonto** [epicmaxco/vuestic-ui#3010]
**Vuestic-ui version:** 1.5.3

**Steps to reproduce**
I want to use toast to display the axios errors but it's not working.

```
Uncaught (in promise) Error: Vuestic GlobalConfigPlugin is not registered
    at useGlobalConfig (global-config.ts:59:13)
    at useComponentConfigProps (use-component-config-props.ts:8:28)
    at customSetup (createProxyComponent.ts:72:29)
    at callWithErrorHandling (runtime-core.esm-bundler.js:173:22)
    at setupStatefulComponent (runtime-core.esm-bundler.js:7265:29)
    at setupComponent (runtime-core.esm-bundler.js:7220:11)
    at mountComponent (runtime-core.esm-bundler.js:5542:13)
    at processComponent (runtime-core.esm-bundler.js:5517:17)
    at patch (runtime-core.esm-bundler.js:5119:21)
    at Object.render2 [as render] (runtime-core.esm-bundler.js:6278:13)
```

Here is my code snippet
```typescript
  private setResponseInterceptor() {
    this.axiosInstance.interceptors.response.use(
      (response: AxiosResponse) => response,
      (error: AxiosError) => {
        if (error.isAxiosError) {
          this.showToast(error.message)
          return Promise.reject(error)
        }

        if (error?.response?.status === 401) {
          this.showToast('Your session has expired. Please login again.')
          useAuthStore().forget()
          useRouter().push({ name: 'Login' })
        } else {
          const response = error?.response?.data as { message: string }
          this.showToast(response.message || 'Something went wrong')
        }
      }
    )
  }

  private showToast(message: string) {
    useToast().init({
      message: message,
      color: 'danger'
    })
  }
```

**Comment by m0ksem** [comment]
Related to https://github.com/epicmaxco/vuestic-ui/issues/3006

**Comment by m0ksem** [comment]
This is actually fixed in 1.6.0.

**Comment by sunrui** [comment]
also cannot work in vuestic-admin?
